raiwidgets/raiwidgets/explanation_dashboard_input.py

```diff
@@ -124,14 +126,17 @@ class and for the regression case a method of predict()
                     "Exceeds maximum number of rows"
                     "for visualization (100000)")
             if feature_length > 1000:
-                raise ValueError("Exceeds maximum number of features for"
-                                 " visualization (1000). Please regenerate the"
-                                 " explanation using fewer features or"
-                                 " initialize the dashboard without passing a"
-                                 " dataset.")
-            self.dashboard_input[
-                ExplanationDashboardInterface.TRAINING_DATA
-            ] = serialize_json_safe(list_dataset)
+                warnings.warn("Exceeds maximum number of features for"
+                              " visualization (1000)."
+                              " Please regenerate the"
+                              " explanation using fewer features or"
+                              " initialize the dashboard without"
+                              " passing a dataset. Dashboard will"
+                              " show limited view.")
+            else:
+                self.dashboard_input[
+                    ExplanationDashboardInterface.TRAINING_DATA
+                ] = serialize_json_safe(list_dataset)
```

Comment:
Does the dashboard loads with more than 1000 features? What was the reason behind the ValueError exception?

Reply:
no, in this case it will just load the explanation but not the importances, and it will print the warning

Reply:
same as behavior in AML Studio, but widget for some reason was failing.  Customer wanted to see the global importance values.